pkg/scanner/adapter.go

```diff
@@ -23,8 +22,11 @@ func NewAdapter(command aqua.Command, transformer Transformer) Adapter {
 }
 
 func (s *adapter) Scan(req harbor.ScanRequest) (harbor.ScanReport, error) {
-	imageRef := fmt.Sprintf("%s:%s", req.Artifact.Repository, req.Artifact.Tag)
-	aquaScanReport, err := s.command.Exec(imageRef)
+	aquaScanReport, err := s.command.Exec(aqua.ImageRef{
+		Repository: req.Artifact.Repository,
+		Tag:        req.Artifact.Tag,
+		Digest:     req.Artifact.Digest,
```

Comment:
Is this digest always passed?

Reply:
Because when it isn't passed, an image becomes like `alpine@`

Reply:
Good question. Both Tag and Digest are passed from Harbor to the Adatper. Those are mandatory properties validated by HTTP handler. If they're blank the HTTP handler will reply with 422 Unprocessable Entity status code.

Reply:
I see.